frontend/class-frontend.php

```diff
@@ -126,6 +126,7 @@ protected function __construct() {
 			new WPSEO_Frontend_Primary_Category(),
 			new WPSEO_JSON_LD(),
 			new WPSEO_Remove_Reply_To_Com(),
+			new WPSEO_OpenGraph_OEmbed(),
```

Comment:
It's not the most ideal location (now the frontend logic and opengraph logic is getting mixed), but I have no better place in mind.

Reply:
Fair enough, then I will leave it where it is for now, thanks!